bridge-proxy/src/bridge-proxy.rs

```diff
@@ -95,20 +94,6 @@ pub trait BridgeProxyContract:
         tx_call.register_promise();
     }
 
-    #[endpoint(cancel)]
```

Comment:
wouldn't have been better just to remove the `#[endpoint(cancel)]` definition and a TODO here to fix & re-add the marker?